src/components/SearchModal/CurrencyList/CurrencyList.tsx

```diff
@@ -9,20 +9,20 @@ import { FixedSizeList } from 'react-window'
 import { Text } from 'rebass'
```

Comment:
Rename this to `index.tsx`, because its already namespaced by its folder.
The test should also be renamed to `index.test.tsx`.

Reply:
Sounds good done